ui/app/services/path-help.js

```diff
@@ -24,37 +30,252 @@ export default Service.extend({
     });
   },
 
+  getNewModel(modelType, backend, apiPath, itemType) {
```

Comment:
what is `itemType` here? I think a comment with example usage of the method would help too

Reply:
`itemType` is the type of nested item, i.e. a LDAP `group` or userpass `user`. here's an example of where the `itemType` is set in the go code: https://github.com/hashicorp/vault/pull/7513/files#diff-cb2f72de1a3a7afc5227ef80f58f99f9R24

(also lol hi delayed answer here 🙈 -- just marking up this PR for future teammates)